languages/JavaScript/exercises/concept/arrays-basic/.docs/hints.md

```diff
@@ -0,0 +1,23 @@
+### 1. Retrieve a card from a stack
+
+- Note that the array index starts from `0`.
+
+### 2. Exchange a card in the stack
+
+- The array is a mutable structure, you can change its content anytime.
```

Comment:
Maybe it is possible to link to documentation that shows how to do this?

Reply:
Maybe 'Access the Elements of an Array' section from [this](https://www.w3schools.com/js/js_arrays.asp) w3schools page, but I can't find an url which scrolls into that section (no id there).

Reply:
That resource does look nice. The other information in that document is nice too, so maybe still use it and just note that the information is in the "Changing an Array Element" section?

Reply:
Perhaps https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array#Common_operations